torch/testing/_internal/common_methods_invocations.py

```diff
@@ -136,6 +137,7 @@ def __init__(self,
 
         self.test_inplace_grad = test_inplace_grad
         self.test_complex_grad = test_complex_grad
+        self.test_scripted_jit = test_scripted_jit
```

Comment:
We could consider making the var name more clear `test_autodiff` maybe?

Reply:
Maybe `test_scripted_jit_grad`, because it specifically determines whether to test the scripted jit part and all other gradchecks are independent of this flag. And `_grad` suffix is similar to other flags.